sql/core/src/main/scala/org/apache/spark/sql/execution/BaseScriptTransformationExec.scala

```diff
@@ -55,8 +58,11 @@ trait BaseScriptTransformationExec extends UnaryExecNode {
   override def outputPartitioning: Partitioning = child.outputPartitioning
 
   override def doExecute(): RDD[InternalRow] = {
+    val hadoopConf = sqlContext.sessionState.newHadoopConf()
+    hadoopConf.set(SQLConf.SCRIPT_TRANSFORMATION_COMMAND_WRAPPER.key,
```

Comment:
Why should we set in Hadoop configuration?

Reply:
> Why should we set in Hadoop configuration?

Executor side SQLConf  can be same with Session level SQLConf? I have tried a lot use SQLConf it will always get default value.

Reply:
Yes, it dose. It passes the configurations via local properties and executor sides. should be able to access it via `SQLConf.get`.

Reply:
> Yes, it dose. It passes the configurations via local properties to executor sides. should be able to access it via `SQLConf.get`.

This piece of code is not particularly familiar, I'm going to check this way

Reply:
Updated, work well, many thanks.